sale_automatic_workflow/models/automatic_workflow_job.py

```diff
@@ -38,7 +38,10 @@ def _validate_sale_orders(self, order_filter):
         _logger.debug('Sale Orders to validate: %s', sales.ids)
         for sale in sales:
             with savepoint(self.env.cr):
+                user_company = self.env.user.company_id
```

Comment:
@lmignon but the problem I pointed out still exists with this? Isn't there a module your company made for having a user to make all these things?